controlplane/kubeadm/controllers/kubeadm_control_plane_controller.go

```diff
@@ -75,6 +75,8 @@ type KubeadmControlPlaneReconciler struct {
 	recorder   record.EventRecorder
 
 	remoteClientGetter remote.ClusterClientGetter
+
+	ManagementCluster *internal.ManagementCluster
```

Comment:
add godoc given that's exported

Reply:
shouldn't be exported, thanks for the catch